scalyr_agent/builtin_monitors/redis_monitor.py

```diff
@@ -187,6 +195,65 @@ def log_slowlog_entries(self, logger, lines_to_fetch):
         for entry in reversed(unseen_entries):
             self.log_entry(logger, entry)
 
+    def log_cluster_replication_info(self, logger):
+        # Gather replication information from Redis
+        replication_info = parse_info(self.redis.execute_command("INFO REPLICATION"))
+
+        # We only care about information from Redis master, which contains offsets from both master and replica
+        if replication_info["role"] != "master":
+            return
+
+        master_repl_offset = replication_info["master_repl_offset"]
+
+        if master_repl_offset == 0:
+            return
+
+        master_replid = replication_info["master_replid"]
+        connected_replicas = replication_info["connected_slaves"]
+
+        # If there are more than one replicas, log the most up-to-date
+        max_replica_offset = 0
+        for n in range(connected_replicas):
+            max_replica_offset = max(
+                max_replica_offset, replication_info["slave{}".format(n)]["offset"]
```

Comment:
For consistency reasons with the rest of the agent, can you use `%` formatting instead of `.format`?